types/react-modal/react-modal-tests.tsx

```diff
@@ -64,6 +64,7 @@ class ExampleOfUsingReactModal extends React.Component {
     };
     return (
       <ReactModal
+        id={"test-id"}
```

Comment:
Thanks for your contribution!

It's a small nit, but like most of the code I read and write (including this very file), the authors forgo the curly braces when assigning a (non-template) string value as a prop to a react component. We should also do that here for consistency within the file.

Also, adding the value `"test-id"` for the `id` prop when we already have a `testId` prop seems kind of backward. Maybe we can swap those two values?

Thanks again!

```suggestion
        id="test-id"
```

Reply:
Hey, thanks for your review :)

I removed the curly braces and changed id's value to "modal-id"

👍